docs/usage.md

```diff
@@ -20,6 +20,15 @@ If you want to configure _deptry_ using `pyproject.toml`, or if your dependencie
 
 ## Dependencies extraction
 
+_deptry_ extracts dependencies into 2 separate groups:
+
+- "production" ones, meant to be used in the codebase
```

Comment:
Not necessarily a huge fan of "production" word, but I did not find a better way to name those dependencies (and having only "dependencies" would have been a bit weird). If you have a better wording, feel free to suggest changes.